src/lightning/fabric/fabric.py

```diff
@@ -745,6 +749,18 @@ def load(
                 state[k] = unwrapped_state[k]
         return remainder
 
+    # Name proposals:
+    # load_raw
+    # load_object
+    # load_torch
+    # load_from_torch
+    # load_non_fabric
+    # load_state_dict (?)
+    def load_object(self, path, obj: Any):
```

Comment:
Proposed alternative names:
load_raw
load_object
load_torch
load_from_torch
load_non_fabric

Personal opinion is that this feature should be combined with `fabric.load()`.